k8s/staging/ingress-patch.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
this is a patch to the ingress that is not really used but is needed to issue the certificate